src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin watch-hands-position {
+  position: absolute;
+  top: ($watch-size / 10 * 2);
+  left: ($watch-size / 2) - ($small-hand-width / 2);
```

Comment:
Maybe better accept some arguments in mixin to set proper values for top & bottom here? Then you could omit to overwrite them for a minute hand.